modules/nw-metallb-frr-k8s-configuration-crd.adoc

```diff
@@ -208,6 +208,55 @@ spec:
     foo: "bar"
 ----
 
+[id="nw-metallb-frrconfiguration-crd-interface_{context}"]
+== The interface field
+
+You can use the  `interface` field to dynamically discover IPv6 neighbors and create a direct connection without the need for IP addresses. 
```

Comment:
I would avoid using the "dynamic"  word, and use the "unnumbered BGP" which a better reference for searching  the web, and is more occurate as we  do BGP over intefaces which do not have IP address (like unnumbered stands for  interface without ip address)

Reply:
Thanks. I've updated.